libminifi/include/utils/file/FileManager.h

```diff
@@ -72,12 +76,12 @@ class FileManager {
         unique_files_.push_back(file_name);
       return file_name;
     } else {
-	  std::string tmpDir = "/tmp";
-	  #ifdef WIN32
-			TCHAR lpTempPathBuffer[MAX_PATH];
-			GetTempPath(MAX_PATH, lpTempPathBuffer);
-			tmpDir = lpTempPathBuffer;
-	  #endif
+    std::string tmpDir = "/tmp";
+    #ifdef WIN32
+      TCHAR lpTempPathBuffer[MAX_PATH];
+      GetTempPath(MAX_PATH, lpTempPathBuffer);
+      tmpDir = lpTempPathBuffer;
+    #endif
```

Comment:
The indentation is still incorrect. Your editor might have got confused because of the preprocessor checks

Reply:
Corrected.